niceman/interface/create.py

```diff
@@ -73,69 +99,26 @@ class Create(Interface):
             doc="Name or ID of the resource to clone to another new resource",
             constraints=EnsureStr(),
         ),
-        image=Parameter(
-            args=("-i", "--image",),
-            doc="Base image ID from which to create the running instance",
-            constraints=EnsureStr(),
-        ),
-        docker_engine_url=Parameter(
-            args=("--docker-engine-url",),
-            doc="URL where Docker engine is listening for connections",
-            constraints=EnsureStr(),
-        ),
         only_env=Parameter(
             args=("--only-env",),
             doc="only env spec",
             nargs="+",
             #action="store_true",
         ),
-        aws_access_key_id=Parameter(
-            args=("--aws-access-key-id",),
-            doc="AWS access key for remote access to your Amazon subscription.",
-            constraints=EnsureStr(),
-        ),
-        aws_secret_access_key=Parameter(
-            args=("--aws-secret-access-key",),
-            doc="AWS secret access key for remote access to your Amazon subscription",
-            constraints=EnsureStr(),
-        ),
-        aws_instance_type=Parameter(
-            args=("--aws-instance-type",),
-            doc="The type of Amazon EC2 instance to run. (e.g. t2.medium)",
-            constraints=EnsureStr(),
-        ),
-        aws_security_group=Parameter(
-            args=("--aws-security-group",),
-            doc="The Amazon security group to assign to the EC2 instance.",
-            constraints=EnsureStr(),
-        ),
-        aws_region_name=Parameter(
-            args=("--aws-region-name",),
-            doc="The Amazon availability zone to run the EC2 instance in. (e.g. us-east-1)",
-            constraints=EnsureStr(),
-        ),
-        aws_key_name=Parameter(
-            args=("--aws-key-name",),
-            doc="Name of SSH key-pair registered in your AWS subscription.",
-            constraints=EnsureStr(),
-        ),
-        aws_key_filename=Parameter(
-            args=("--aws-key-filename",),
-            doc="Path to SSH private key file matched with AWS key name parameter.",
-            constraints=EnsureStr(),
-        ),
         existing=Parameter(
             args=("-e", "--existing"),
             choices=("fail", "redefine"),
             doc="Action to take if name is already known"
         ),
+        backend=Parameter(
+            args=("-b", "--backend"),
```

Comment:
I think we don't really need it as an additional option and "create" could just take then as arguments (just avoid typing `--backend` pretty much).  but ok for now -- we will RF later.

Cheers